.circleci/config.yml

```diff
@@ -47,18 +46,40 @@ jobs:
       - run:
           name: Build Etherpot Prime SDK
           command: npm run build
+      - run:
+          name: Checkout e2e repo and run tests
+          no_output_timeout: 60m
+          command: |
+            cd ~
+            git clone https://github.com/etherspot/e2e-sdk-prime.git
+            cd e2e-sdk-prime
+            sed -i.bak "s/_private_key_/$PRIVATE_KEY/g" .env
+            mv package.json temp.json
+            jq -r '.dependencies."@etherspot/prime-sdk" |= "file:../etherspot-prime-sdk"' temp.json > package.json
+            rm temp.json
+            npm i
+            touch ~/logs.txt
+            npm run test-mainnet |& tee ~/logs.txt
+            chmod 755 ~/logs.txt
+            cat ~/logs.txt
+            echo "export LOGS='$(cat ~/logs.txt)'" >> $BASH_ENV
+            echo $BASH_ENV
+            rm ~/logs.txt
+            source "$BASH_ENV"
       - run:
           name: Publish package to npm
-          command: npm publish --access=public
+          command: |
+            cd ~/etherspot-sdk
+            npm publish
       - run:
           name: Announce Publish
           command: |
             chmod +x .circleci/announcePublish.sh
             .circleci/announcePublish.sh "Etherspot Prime SDK" "$(node -e "console.log(require('./package.json').version)")"
       - slack/status:
-          fail_only: true
           failure_message: "Triggered by: *${CIRCLE_USERNAME}* \n\n Ooops! The *$CIRCLE_JOB* job has failed! :circleci-fail:"
-          only_for_branches: master
+          success_message: "Triggered by: *${CIRCLE_USERNAME}* \n\n Woohoh! The *$CIRCLE_JOB* job has suceeded! :circleci-pass: \n\n $LOGS"
```

Comment:
Can we please have specific message here for e2e tests?
e.g "e2e tests completed successfully" or "e2e tests failed"